django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
And if you don't have `vary:x-requested-with` then ATS will think that the ajax and normal requests are the same and cache and mix them up. But ajax requests really should have their own separate url instead.